bundle/edu.gemini.p1backend/TODO.md

```diff
@@ -0,0 +1,50 @@
+# Short term
+
+Something that improves the situation now. Nothing for the future. A dead-end by
+design. Something to keep chugging away until we do it right. It shouldn't
+take more work than 2 weeks.
+
+1. Code lives in ocs.
+1. URLs:
+   - Receiver of P1 submissions: phase1.gemini.edu/receive/.../...
+   - Viewer of P1 submissions: phase1.gemini.edu/fetch/.../...
+1. Each backend a different path like now.
+1. Configuration file for each backend (that file is read with every new
```

Comment:
The idea was to increment the counter only after making sure everything was right, the file is stored to disk with the right Id, the PDFs have been created, the emails have been sent, etc.

But now that you mention it there may be edge cases where we would still need a rollback regardless. Which makes me wonder even more why the counter is so necessary.